src/phases/execution/LookupCommands.yml

```diff
@@ -0,0 +1,140 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query-execution"
+Description: |
+  Defines common configurations for 'LookupSBEPushdown' and 'LookupSBEPushdownMisc' workloads.
+
+# Defines value generators.
+
+# Generates a random integer value from 1 to 'card' parameter of which the default is 9500.
+RandIntUpToTemplate: &RandIntUpTo {^RandomInt: {min: 1, max: {^Parameter: {Name: "card", Default: 9500}}}}
+# Generates a monotonically increasing integer value from 1 to 'card' parameter of which the default
+# is 9500.
+IncIntUpToTemplate: &IncIntUpTo {^Inc: {start: 1, end: {^Parameter: {Name: "card", Default: 9500}}}}
```

Comment:
I don't think the ^Inc generator has an end field. We could add one. What should it do when it gets there? Should it wrap around back to start? https://github.com/mongodb/genny/blob/2ed9a81c20ef0f4a76420f67f94017a9b3dd8365/src/value_generators/src/DocumentGenerator.cpp#L1017